.github/workflows/configurations.yml

```diff
@@ -29,8 +29,8 @@ jobs:
         if: env.DOCKER_USR != ''
         with:
           registry: registry.upbound.io
-          username: ${{ secrets.UPBOUND_ROBOT_USR }}
-          password: ${{ secrets.UPBOUND_ROBOT_PSW }}
+          username: ${{ secrets.UPBOUND_XP_ROBOT_USR }}
+          password: ${{ secrets.UPBOUND_XP_ROBOT_PSW }}
```

Comment:
Would it make sense to just use the existing xp org for everything? Both Crossplane itself, as well as configurations (and in future providers)? Having two orgs that both mean kind of the same thing ("crossplane" and abbreviation of "crossplane") seems a little confusing.

Reply:
Just noting here as we discussed offline: `crossplane` org already exists as well and that is where providers are manually synced today, so we want to keep using it. We may opt to consolidate in the future, but for now `crossplane` org = offical and `xp` org = supplementary artifacts (experimental / docs / etc.)